src/machine/machinectl.c

```diff
@@ -533,7 +533,11 @@ static void print_machine_status_info(sd_bus *bus, MachineStatusInfo *i) {
                 putchar('\n');
 
         s1 = strdup(strempty(FORMAT_TIMESTAMP_RELATIVE(i->timestamp.realtime)));
+        if (!s1)
+                return log_oom();
         s2 = strdup(strempty(FORMAT_TIMESTAMP(i->timestamp.realtime)));
```

Comment:
Hmm, there's no need to do memory allocation here at all? Also, the code below handles OOM gracefully

Reply:
> Also, the code below handles OOM gracefully

Right, sorry for the noise.

Reply:
(could you remove memory allocation here?)